server/server.js

```diff
@@ -6,7 +6,7 @@ const app = express();
 
 app.use(express.static('./public/dist'));
 
-app.get('/:id', (req, res) => {
+app.get('/reviews/:id', (req, res) => {
   if (req.params.id !== 'favicon.ico') {
```

Comment:
This all looks good. Just informationally - do you have to do that favicon.ico check on an API path? (I've never done that, but it seems like an easy thing to check)